src/content/reference/hi/p5/loadModel.mdx

```diff
@@ -1,91 +1,34 @@
 
-  <p>The first parameter, <code>path</code>, is always a <code>String</code>
-  with the path to the
+  <p>कॉल करने का दूसरा तरीका <code>loadModel()</code> में फ़ाइल पथ के बाद चार वैकल्पिक पैरामीटर हैं।
+  
+  पहला वैकल्पिक पैरामीटर एक <code>boolean</code> मान है। यदि <code>true</code> पास हो जाता है, जैसे कि <code>loadModel('/assets/model.obj', true)</code>, तो यह सुनिश्चित करने के लिए मॉडल का आकार बदल दिया जाएगा कि यह कैनवास पर फिट बैठता है।
+  
+  अगले तीन पैरामीटर हैं <code>successCallback</code>, <code>failureCallback</code>, और <code>fileType</code> जैसा कि ऊपर बताया गया है।</p>
 
-  file. Paths to local files should be relative, as in
-
-  <code>loadModel('/assets/model.obj'). URLs such as </code>'<a
-  href="https://example.com/model.obj'%60">https://example.com/model.obj'`</a>
-  may be blocked due to browser security.</p>
-
-  <p>The first way to call <code>loadModel()</code> has three optional
-  parameters after the
-
-  file path. The first optional parameter, <code>successCallback</code>, is a
-  function
-
-  to call once the model loads. For example,
-
-  <code>loadModel('/assets/model.obj', handleModel)</code> will call the
-  <code>handleModel()</code>
-
-  function once the model loads. The second optional parameter,
-
-  <code>failureCallback</code>, is a function to call if the model fails to
-  load. For
-
-  example, <code>loadModel('/assets/model.obj', handleModel,
-  handleFailure)</code> will
-
-  call the <code>handleFailure()</code> function if an error occurs while
-  loading. The
-
-  third optional parameter, <code>fileType</code>, is the model’s file extension
-  as a
-
-  string. For example,
-
-  <code>loadModel('/assets/model', handleModel, handleFailure, '.obj')</code>
-  will try to
-
-  load the file model as a <code>.obj</code> file.</p>
-
-  <p>The second way to call <code>loadModel()</code> has four optional
-  parameters after the
-
-  file path. The first optional parameter is a <code>Boolean</code> value. If
-  <code>true</code> is
-
-  passed, as in <code>loadModel('/assets/model.obj', true)</code>, then the
-  model will be
-
-  resized to ensure it fits the canvas. The next three parameters are
-
-  <code>successCallback</code>, <code>failureCallback</code>, and
-  <code>fileType</code> as described above.</p>
-
-  <p>The third way to call <code>loadModel()</code> has one optional parameter
-  after the
-
-  file path. The optional parameter, <code>options</code>, is an
-  <code>Object</code> with options,
-
-  as in <code>loadModel('/assets/model.obj', options)</code>. The
-  <code>options</code> object can
-
-  have the following properties:</p>
+  <p>कॉल करने का तीसरा तरीका <code>loadModel()</code> में फ़ाइल पथ के बाद एक वैकल्पिक पैरामीटर है। वैकल्पिक पैरामीटर, <code>options</code>, विकल्पों के साथ एक <code>object</code> है, जैसे <code>लोडमॉडल('/assets/model.obj', options)</code> में। <code>options</code> ऑब्जेक्ट में निम्नलिखित गुण हो सकते हैं:</p>
```

Comment:
Do not translate loadModel in the code tag

Reply:
Thanks!